Assets/Scripts/ProjController.cs

```diff
@@ -0,0 +1,66 @@
+using UnityEngine;
+
+public class ProjController : MonoBehaviour
+{
+    [SerializeField]
+    private int projDamage = 100;
+    [SerializeField]
+    private float projMoveSpeed = 7f;
+    [SerializeField]
+    private bool wantDeviation = true;
+    [SerializeField]
+    private float projDeviation = 1f;
+
+    public string target = "null"; // Needs to be set on creation by creating object, "Player" to damage players, "Enemy" to damage enemies
```

Comment:
An empty string should be used instead of the string `"null"` because it is confusing to use that when a string can have a value of `null`.